includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -1071,14 +1068,38 @@ public function preview() {
 		}
 	}
 
+	/**
+	 * Helper which formats the scheduled date and sets the appropriate dates in the job data.
+	 *
+	 * @param array  $job_data       The job data array to modify.
+	 * @param string $scheduled_date The scheduled date.
+	 *
+	 * @return bool True when the scheduled date is a valid future date, false otherwise.
+	 */
+	public static function calculate_scheduled_dates( array &$job_data, string $scheduled_date ): bool {
```

Comment:
Would something like `apply_scheduled_date` or `set_scheduled_date` be a more appropriate name for this?

Reply:
Yep much better